recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ build:
   number: 3
   script: |
     {{ PYTHON }} -m pip install . -vv
-  skip: True  # [win or (aarch64 and py<39)]
+  skip: True  # [win]
```

Comment:
Also will need to [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks )